Description: Identify newer python-sphinx versions
 In recent versions, "sphinx-build --version" changed its output format
 and moved from STDOUT to STDERR.
Author: Nicholas Breen <nbreen@debian.org>
Origin: vendor
Last-Update: 2018-04-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: gromacs/cmake/FindSphinx.cmake
===================================================================
--- gromacs.orig/cmake/FindSphinx.cmake
+++ gromacs/cmake/FindSphinx.cmake
@@ -44,12 +44,13 @@ mark_as_advanced(SPHINX_EXECUTABLE)
 if (SPHINX_EXECUTABLE AND NOT DEFINED SPHINX_EXECUTABLE_VERSION)
     execute_process(
         COMMAND ${SPHINX_EXECUTABLE} --version
+        ERROR_VARIABLE  SPHINX_VERSION_OUTPUT_VARIABLE
         OUTPUT_VARIABLE SPHINX_VERSION_OUTPUT_VARIABLE
         RESULT_VARIABLE SPHINX_VERSION_RESULT_VARIABLE
         ERROR_QUIET
         OUTPUT_STRIP_TRAILING_WHITESPACE
         )
-    string(REGEX REPLACE "Sphinx \\([^)]*\\) ([^ ]+)" "\\1" SPHINX_EXECUTABLE_VERSION "${SPHINX_VERSION_OUTPUT_VARIABLE}")
+    string(REGEX MATCH "[0-9]+\\.[0-9]+\\.[0-9]+" SPHINX_EXECUTABLE_VERSION "${SPHINX_VERSION_OUTPUT_VARIABLE}")
     set(SPHINX_EXECUTABLE_VERSION "${SPHINX_EXECUTABLE_VERSION}" CACHE INTERNAL "Version of ${SPHINX_EXECUTABLE}")
 endif()
 
